### ISSUE TITLE
Question: supported Android devices/browsers?

### ISSUE THREAD

**Issue by Dissimilis** [fido-alliance/webauthn-demo#5]
You demo works perfectly with desktop Chrome and FF.
Is there any way to test this on Android? I tried Chrome Canary and FF nightly with no luck. Does any combination of Android + Browser work for testing WebAuthn?

**Comment by herrjemand** [comment]
@Dissimilis This should work on the latest versions of chrome for android. And i am not sure about canary and nightly. What versions have you tried?

**Comment by whitehatguy** [comment]
I was able to plug in a Yubico key to a Nexus 5X using a dongle and was able to utilize it.  The user interface was a bit odd - Android / Chrome asked to set-up my key each and every time I invoked the log-in process; that workflow required approving "Google Play Services" access to my USB device; it was also a bit touchy as to the timing (sometimes it just wouldn't work.)  I'm not sure if this is a problem with the UX of Android / Chrome, or if it is particular to the way in which webauthn.org demo is invoking the APIs?  This is Chrome build 70.0.3538.80 on Android 8.1.
